pandas/core/window/expanding.py

```diff
@@ -96,7 +96,7 @@ class Expanding(RollingAndExpandingMixin):
     4  7.0
     """
 
-    _attributes = ["min_periods", "center", "axis", "method"]
+    _attributes: list[str] = ["min_periods", "center", "axis", "method"]
```

Comment:
isn't `mypy` able to infer this?

Reply:
I thought so, but I got:
pandas/core/window/expanding.py:676: error: Cannot determine type of "_attributes"  [has-type]
pandas/core/window/ewm.py:706: error: Cannot determine type of "_attributes"  [has-type]

I then annotated `_attributes` of the super-classes (the sub-classes concatenate the attributes of the super-classes).